symfony.lock

```diff
@@ -525,274 +271,179 @@
+            "ref": "673836afb0eac2b0ec36c44f2ff0379e5a4b2177"
         }
     },
-    "symfony/password-hasher": {
-        "version": "v5.3.2"
-    },
-    "symfony/polyfill-mbstring": {
-        "version": "v1.9.0"
-    },
-    "symfony/polyfill-php81": {
-        "version": "v1.23.0"
-    },
-    "symfony/process": {
-        "version": "v4.1.3"
-    },
-    "symfony/property-access": {
-        "version": "v4.1.3"
-    },
-    "symfony/property-info": {
-        "version": "v5.1.2"
-    },
-    "symfony/proxy-manager-bridge": {
-        "version": "v4.1.3"
-    },
-    "symfony/redis-messenger": {
-        "version": "v5.2.1"
-    },
     "symfony/routing": {
-        "version": "4.0",
+        "version": "7.1",
         "recipe": {
             "repo": "github.com/symfony/recipes",
-            "branch": "master",
-            "version": "4.0",
-            "ref": "cda8b550123383d25827705d05a42acf6819fe4e"
-        }
+            "branch": "main",
+            "version": "7.0",
+            "ref": "21b72649d5622d8f7da329ffb5afb232a023619d"
+        },
+        "files": [
+            "config/packages/routing.yaml",
+            "config/routes.yaml"
+        ]
     },
     "symfony/security-bundle": {
-        "version": "3.3",
+        "version": "7.1",
         "recipe": {
             "repo": "github.com/symfony/recipes",
-            "branch": "master",
-            "version": "3.3",
-            "ref": "f8a63faa0d9521526499c0a8f403c9964ecb0527"
-        }
-    },
-    "symfony/security-core": {
-        "version": "v4.4.18"
-    },
-    "symfony/security-csrf": {
-        "version": "v5.2.1"
-    },
-    "symfony/security-guard": {
-        "version": "v4.4.18"
-    },
-    "symfony/security-http": {
-        "version": "v4.4.18"
-    },
-    "symfony/serializer": {
-        "version": "v5.1.2"
-    },
-    "symfony/service-contracts": {
-        "version": "v1.1.6"
+            "branch": "main",
+            "version": "6.4",
+            "ref": "2ae08430db28c8eb4476605894296c82a642028f"
+        },
+        "files": [
+            "config/packages/security.yaml",
+            "config/routes/security.yaml"
+        ]
     },
     "symfony/stimulus-bundle": {
-        "version": "2.19",
+        "version": "2.20",
         "recipe": {
             "repo": "github.com/symfony/recipes",
             "branch": "main",
-            "version": "2.8",
-            "ref": "9e33a8a3794b603fb4be6c04ee5ecab901ce549e"
-        }
-    },
-    "symfony/stopwatch": {
-        "version": "v4.1.3"
-    },
-    "symfony/string": {
-        "version": "v5.2.1"
-    },
-    "symfony/templating": {
-        "version": "v4.1.3"
-    },
-    "symfony/thanks": {
-        "version": "v1.0.8"
+            "version": "2.13",
+            "ref": "6acd9ff4f7fd5626d2962109bd4ebab351d43c43"
+        },
+        "files": [
+            "assets/bootstrap.js",
+            "assets/controllers.json",
+            "assets/controllers/hello_controller.js"
```

Comment:
I was unsure how to handle these files and decided to remove them, as the way we have configured our UI seems to follow a different architecture